kornia/augmentation/container/image.py

```diff
@@ -71,24 +79,19 @@ def __init__(
         return_transform: Optional[bool] = None,
         keepdim: Optional[bool] = None,
         random_apply: Union[int, bool, Tuple[int, int]] = False,
+        validate_args: bool = True,
     ) -> None:
-        self.same_on_batch = same_on_batch
-        self.return_transform = return_transform
-        self.keepdim = keepdim
+        self._same_on_batch = same_on_batch
+        self._return_transform = return_transform
+        self._keepdim = keepdim
         # To name the modules properly
         _args = OrderedDict()
         for idx, arg in enumerate(args):
```

Comment:
```suggestion
        for idx, mod in enumerate(args):
```
we iterate modules here only?

Reply:
yes